requirements/constraints.txt

```diff
@@ -14,7 +14,13 @@
 
 # As it is not clarified what exact breaking changes will be introduced as per
 # the next major release, ensure the installed version is within boundaries.
-celery>=5.2.2,<6.0.0
+# celery>=5.2.2,<6.0.0
+
+# 5.3.0 versions breaking timezone related tests. Adding pin for time being to clear upgrade pipeline. Unpin above
+# after fixing these failures.
```

Comment:
Let's create an issue on the board for unpinning this constraint and mention the issue link in the file for others.